aptos-move/block-executor/src/proptest_types/types.rs

```diff
@@ -153,26 +202,42 @@ impl Arbitrary for ValueType {
 }
 
 impl ValueType {
-    /// If use_value is not set, the resulting Value will correspond to a deletion, i.e.
-    /// not contain a value (o.w. deletion).
+    /// If deletion is false, we use WriteOpKind::Creation by default, but for resource groups
+    /// mock executor updates the WriteOp kind to avoid the consistency check with existence
+    /// (not checked for normal resources, storage asserts).
     pub(crate) fn new<V: Into<Vec<u8>> + Debug + Clone + Eq + Send + Sync + Arbitrary>(
         value: V,
-        use_value: bool,
+        deletion: bool,
     ) -> Self {
         Self {
-            bytes: use_value.then(|| {
+            bytes: deletion.then(|| {
```

Comment:
Earlier this closure is executed when use_value is set to true (deletion is set to false). Now, this closure will execute when deletion is set to true instead. Is that intentional?

Reply:
uh, don't know when this happened, should never have been deletion - it's fixed now.